index.js

```diff
@@ -79,18 +79,14 @@ module.exports = {
     "no-trailing-spaces": 1,
     "no-useless-constructor": 1,
     "one-var": [ 1, { "uninitialized": "always", "initialized": "never" } ],
-
-    /* "object-curly-spacing": [ 2, "never" ], */
-
+    "object-curly-spacing": [ 1, "never" ],
     "quotes": [ 1, "single", {"avoidEscape": true, "allowTemplateLiterals": true} ],
     "semi-spacing": [ 2, { "before": false, "after": true } ],
     "no-multiple-empty-lines": [ 1, { "max": 2 } ],
     "no-extra-semi": 1,
     "space-before-blocks": [ 1, "always" ],
     "space-before-function-paren": [ 1, { "anonymous": "always", "named": "never" } ],
-
-    /* "space-in-parens": [ 2, "always", { "exceptions": ["{}"] } ], */
-
+    "space-in-parens": [ 1, "never" ],
```

Comment:
I've come around to better syntax coloring of parens being better than padding with spaces.